sycl/include/sycl/ext/oneapi/experimental/graph.hpp

```diff
@@ -142,9 +144,14 @@ struct graph_impl {
     } else {
       this->add_root(nodeImpl);
     }
+    MLastNode = nodeImpl;
     return nodeImpl;
   }
 
+  node_ptr getLastNode() const{
```

Comment:
formatting nitpick - should be a space between `const {`

Reply:
Fixed thanks!